WebInkEnhancement/explainer.md

```diff
@@ -150,5 +155,9 @@ Due to uncertainty around the correct execution when `setLastRenderedPoint` is c
 
 Instead of providing `setLastRenderedPoint` with a PointerEvent, just providing x and y values is also an option. It was decided that a trusted pointer event would likely be the better option though, as then we can have easier access to the pointer ID and the web developer doesn't have to put extra thought into the position of the ink.
 
+Providing the presenter with the canvas allows the boundaries of the drawing area to be determined. This is necessary so that points and ink outside of the desired area aren't drawn when points are being forwarded. If no canvas is provided, then the root layer size will be used.

```

Comment:
At this point, I think we need another section/set of paragraphs above Other Considerations that explains the interface in more detail. Putting it here buries it a bit too much.

Reply:
Agreed, added an "Interface Details" section.